sunflower-land/sunflower-land
# [FEAT] Wishing Well Rework

# Description
This PR is a rework of the wishing well flow. The aim is to make the process clearer and more accessible for players. Technically, there has been rework of the wishing well machine which is now a child machine of the goblin machine. 

I am happy to take feedback on the copy in any of the following modals.

<img width="300" alt="Screen Shot 2022-05-21 at 10 53 54 pm" src="https://user-images.githubusercontent.com/17863697/169653118-dd50ffed-05e2-4c06-b3b3-4e031097256b.png">

<img width="300" alt="Screen Shot 2022-05-21 at 10 57 09 pm" src="https://user-images.githubusercontent.com/17863697/169653221-36ea9b83-7cc4-40c9-bd69-3cc82cacfe12.png">

<img width="300" alt="Screen Shot 2022-05-21 at 10 46 44 pm" src="https://user-images.githubusercontent.com/17863697/169653231-83985bb8-a552-436c-b955-099459b7305a.png">

<img width="300" alt="Screen Shot 2022-05-21 at 10 35 04 pm" src="https://user-images.githubusercontent.com/17863697/169653246-0e6b283d-e5fc-4e88-a6cb-634a0497b615.png">

<img width="406" alt="Screen Shot 2022-05-21 at 11 17 07 pm" src="https://user-images.githubusercontent.com/17863697/169653350-8f781643-fb88-4154-b899-4905776e888b.png">

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #issue

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

*To test*
- Go to Goblin Town
- Click on Wishing Well
- Check all links are working
- Add testnet LP tokens
- Close modal
- Open modal and check that you see the timer modal
- After 3 minutes open the Wishing Well again
- Grant Wish
- Check congratulations modal

# Checklist:
